packages/paste-core/components/side-panel/src/SidePanelFooter.tsx

```diff
@@ -11,7 +11,7 @@ const SidePanelFooter = React.forwardRef<HTMLDivElement, SidePanelFooterProps>(
         paddingX={variant === "chat" ? "space50" : "space70"}
```

Comment:
suggestion: I think you're missing the updated background color from the doc

```suggestion
        paddingX={variant === "chat" ? "space50" : "space70"}
        backgroundColor="colorBackgroundWeaker"
```

Reply:
This should of also been done in `SidePanelHeader.tsx` based on the doc

Reply:
are you talking about `$color-background-weaker` in doc? If yes, I added that in line 12, `backgroundColor="colorBackgroundWeaker"`

Reply:
so I talked with @roxanagomez about this and in figma, designers need to add the background color to components like header and footer since they are separate components 
We dont need to do it in code, it will be transparent for footer and header and inherit colors of main body